guides/source/association_basics.md

```diff
@@ -3007,6 +3007,7 @@ With this definition complete, our `Entry` delegator now provides the following
 
 | Method | Return |
 |---|---|
+| `Entry#entryable_types` | ["Message", "Comment"] |
```

Comment:
Should this be `Entry.entryable_types`?

Reply:
Yes indeed.

Reply:
#50819 @tanaken0515 fixed this already - thanks!